database/schedule/list_active_test.go

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2023 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package schedule
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/go-vela/types/library"
+)
+
+func TestSchedule_Engine_ListActiveSchedules(t *testing.T) {
```

Comment:
Very minor, but I would like to see either the test for `ListActiveSchedules` or `CountActiveSchedules` actually leverage the filter by setting one of the schedules to inactive.